elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -823,6 +898,28 @@ class PfeNavigation extends PFElement {
     }
   }
 
+  /**
+   * If lang is not set to "en", translate strings based on object defined in constructor
+   */
+  _translateStrings(lang) {
```

Comment:
@TylerMartin5 it looks like it's translating even if it is set to en? There's no if statement wrapping `this._translateStrings()` and I don't see anything in this function either?

Reply:
@wesruv I think I forgot to update that comment, I'll do that now. I'm not going to add that logic. If "en" is ever passed to the lang attribute it should just read from the object